### DISCUSSION TITLE
[Unstable] Outpost guards attack the infected NPC in 'youngcultists' event.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14675] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

A few moments after the event spawns on the station, the outpost security guards & outpost watchman will be aggroed, drawing their weapons and running into the church of husk room & attacking the infected unconscious churchmember.

Something to do with changed husk affliction logic or changed outpost AI ?

Funny enough, even though the cultist is quickly shot to death - the quest script can still be triggered and proceeds as normal.

### Reproduction steps

- Arrive at a husk station with the 'youngcultists' event active.
- Wait a few moments for the husk infection to progress to it's convulsion stage.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by DownloadableContent5021** [comment]
I also found this bug when playing on MacOS, and I believe this happens with ANY outpost event that involves an unconscious person, because this also happened to me on the event where you find a guy who had overdosed on opium.
It appears to be because the game is considering these unconscious people to be hostiles, because their name was red
<img width="1680" alt="Screenshot 2024-09-23 at 11 27 14 AM" src="https://github.com/user-attachments/assets/6978bbf4-c020-49a0-b70c-a5f385a7d8aa">

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in the next unstable build. Sorry for the inconvenience!
